tests/integration/mongodb/test_charm.py

```diff
@@ -378,11 +379,20 @@ async def test_log_rotate(ops_test: OpsTest, substrate: Substrate, application_p
     log_not_rotated = "audit.log.1" not in log_files
     assert log_not_rotated, f"Found rotated log in {log_files}"
 
-    await start_continous_writes(ops_test, client_app_name=application_name)
-    time.sleep(time_to_write_50m_of_data)
-    await stop_continous_writes(ops_test, client_app_name=application_name)
+    for i in range(5):
```

Comment:
could you add a small comment here - to explain why we are iterating at all (future readers will not understand unless we tell them it's to speed up writes)